crates/lib/src/command/supervisor.rs

```diff
@@ -81,9 +81,6 @@ impl SupervisorBuilder {
 			post_spawn_handler,
 		)
 	}
-}
-
-impl Supervisor {
 	/// Spawns the command set, the supervision task with a random SupervisorId and returns a new control object.
 	pub fn spawn(
```

Comment:
Spawn can be "reimplemented" as

```rust
Self::new(SupervisorArgs {
	errors,
	events,
	commands,
	supervisor_id: Default::default(),
	grouped,
	actioned_events,
	pre_spawn_handler,
	post_spawn_handler,
})
```

and `spawn_with_id` then folded back into `new()`

Reply:
oh, true! that is a lot cleaner !!